modules/nlu-testing/src/tools/visualisation.ts

```diff
@@ -0,0 +1,189 @@
+import * as sdk from 'botpress/sdk'
+import _ from 'lodash'
+import { distance, similarity } from 'ml-distance'
+import { Matrix } from 'ml-matrix'
+import { PCA } from 'ml-pca'
+
+import { BotState } from '../backend/typings'
+const clustering = require('density-clustering')
```

Comment:
import

Reply:
Still curious, of why you can't import this. I'll take a look.